.yamato/_editor_priming.yml

```diff
@@ -16,7 +16,25 @@ editor:priming:trunk:macos:
         UPM_REGISTRY: https://artifactory-slo.bf.unity3d.com/artifactory/api/npm/upm-candidates
     commands:
       -  pip install unity-downloader-cli --user --upgrade --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple --upgrade
-      -  unity-downloader-cli -u $CUSTOM_REVISION -o macos --wait --skip-download -c editor -c il2cpp  > unity_revision.txt
```

Comment:
why does this file show lots of instances of this custom-revision line being removed?

Reply:
its not actually removed, just the $CUSTOM_REVISION is changed to trunk, but with git drawing all diffs together its hard to notice (eg here https://github.com/Unity-Technologies/Graphics/pull/1947/files#diff-623283c72d9818c2389e97591ea56c82L75) 
We dont use editor priming for trunk, so this mistake went unnoticed before. The jobs for custom-revision still use the correct variable though.